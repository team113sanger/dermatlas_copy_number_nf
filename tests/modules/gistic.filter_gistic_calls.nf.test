nextflow_process {

    name "Test Process FILTER_GISTIC2_CALLS"
    script "modules/gistic2.nf"
    process "FILTER_GISTIC2_CALLS"

    test("Should run without failures") {

        when {
        params {
            load("$baseDir/tests/testdata/test_params.json")
        }
            process {
                """
                input[0] = file("$baseDir/tests/testdata/leioma_segments.tsv")
                input[1] = file("$baseDir/tests/testdata/all_lesions.conf_95.txt")
                input[2] = file("${params.difficult_regions_file}")
                input[3] = "${params.cohort_prefix}"
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
