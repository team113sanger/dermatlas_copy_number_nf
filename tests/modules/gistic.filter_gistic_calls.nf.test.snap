{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "6509_2707_Leiomyoma_gistic_sample_summary.tsv:md5,7dd3f6b5d8c33a26b112d75ad8e3deb6"
                ],
                "1": [
                    "6509_2707_Leiomyoma_gistic_cohort_summary.tsv:md5,3484929d572979d94f7c493f465c27dc"
                ],
                "cs": [
                    "6509_2707_Leiomyoma_gistic_sample_summary.tsv:md5,7dd3f6b5d8c33a26b112d75ad8e3deb6"
                ],
                "ss": [
                    "6509_2707_Leiomyoma_gistic_cohort_summary.tsv:md5,3484929d572979d94f7c493f465c27dc"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-12-12T15:12:48.62497"
    }
}